lib/scrabble.rb

```diff
@@ -1,11 +1,21 @@
 class Scrabble
+  require 'pry'
 
-  def score(word)
-    1
+  def score("word")
+    skip
```

Comment:
`skip` is something we give to tests. Right now you're returning the variable `skip` from this function, and since `skip` has not been assigned any value it will always evaluate to `nil`.